src/cirrus/lambda_handlers/post_batch.py

```diff
@@ -1,18 +1,19 @@
 import json
 import re
-from typing import Any, Optional, Tuple
+
+from typing import Any
 
 import boto3
 
-from cirrus.lib2.logging import get_task_logger
-from cirrus.lib2.process_payload import ProcessPayload
+from cirrus.lib.logging import get_task_logger
+from cirrus.lib.process_payload import ProcessPayload
 
-logger = get_task_logger("task.post-batch", payload=tuple())
+logger = get_task_logger("task.post-batch", payload=())
 
 BATCH_LOG_GROUP = "/aws/batch/job"
 LOG_CLIENT = boto3.client("logs")
 DEFAULT_ERROR = "UnknownError"
-ERROR_REGEX = re.compile(r"^(?:cirrus\.?lib\.errors\.)?(?:([\.\w]+):)?\s*(.*)")
+ERROR_REGEX = re.compile(r"^(?:([\.\w]+):)?\s*(.*)")
```

Comment:
why this change?

Reply:
This is a special case for `cirrus.lib.errors.InvalidInput` where it will strip the `cirrus.lib.errors` prefix off that error string. This maybe made sense when we were using cirrus-lib for writing tasks and that was the only `InvalidInput` exception. But now we also have `stactask.exceptions.InvalidInput`, which is what we actually want people to use, so I think the special casing here for an exception from an unmaintained lib is silly to maintain moving forward.

Reply:
Note that the flip side of this is `update-state` matching on the exception name to determine if a failed execution was actually invalid. That is based on the `INVALID_EXCEPTIONS` variable in `update_state.py`.

Reply:
👍🏻